build/static/js/main.ff63dd26.js.LICENSE.txt

```diff
@@ -0,0 +1,41 @@
+/*
+object-assign
```

Comment:
You shouldn't have any files from the build directory in git. Add /build to your .gitignore and remove these files.